### ISSUE TITLE
NOTES - EMERGENCY bug for x127/x162 users: header and crisis info duplicated

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#2897]
If a user in Hennepin or Ramsey county uses the EGA screening functionality, then continues on to use the NOTES - EMERGENCY script, the crisis info from the screening is carried over which causes the header of the case note and the 'crisis' field to duplicate. 

Example case note header: ***Emergency app: homelessnesshomelessness***  

The crisis variable needs to be blanked out prior to going into the NOTES - EMERGENCY script around line 396.

**Comment by kaleykobber** [comment]
Yes! Users are reporting this duplication at Ramsey. Thank you for posting this issue.
